content/applications/productivity/iot/devices/screen.rst

```diff
@@ -0,0 +1,100 @@
+================
+Connect a screen
+================
+
+In Odoo, an :abbr:`IoT (Internet of Things)` Box can be connected to a screen display. After being
+connected, the screen can be used to display a :abbr:`PoS (Point of Sale)` order to a client.
+
+.. image:: screen/screen-pos-client-display.png
+   :align: center
+   :alt: An example of a PoS (point of sale) order on a screen display.
+   
+.. note::
+   Access the customer display from any other computer by going to the :abbr:`IoT (Internet of Things)`
+   Box homepage and clicking on the :guilabel:`POS Display` button.
+
+
+Connection
+==========
+
+The way to connect the screen display to the :abbr:`IoT (Internet of Things)` Box differs depending
+on the model.
+
+.. tabs::
+
+   .. tab:: IoT Box model 4
+
+      Connect up to two screens with Micro-HDMI cables on the side of the :abbr:`IoT (Internet of Things)`
+      Box. If two screens are connected, they can display distinct content (see usage below).
+
+   .. tab:: IoT Box model 3
+
+      Connect the screen with an HDMI cable on the side of the :abbr:`IoT (Internet of Things)` Box.
+
+.. important::
+   Screen(s) should be connected before the :abbr:`IoT (Internet of Things)` Box is switched on. If it
+   is already on, connect the screen(s), and then restart the :abbr:`IoT (Internet of Things)` Box by
+   unplugging it and plugging it back into its power source.
+
+.. warning::
+   The usage of HDMI/Micro-HDMI adapters may cause issues which will result in a blank, black screen on
+   the screen display. Cable usage is recommended.
```

Comment:
"Cable usage is recommended"? I'm not sure to understand that. "Correct cable usage" maybe?